docs/references/commands.txt

```diff
@@ -0,0 +1,326 @@
+..
+   THIS DOCUMENTATION WAS AUTOMATICALLY GENERATED
+   Do not edit this document directly. Instead, edit the commands in the cli
+   source code, which can be found on the main branch of the git repo at
+   https://github.com/canonical/ubuntu-pro-client. The cli source code is
+   nested in the uaclient/cli folder. Description for the commands is found
+   in the uaclient/messages folder.
+
+Available commands
+==================
+
+The currently available commands are:
+
+- `pro api`_
+- `pro attach`_
+- `pro auto-attach`_
+- `pro collect-logs`_
+- `pro config`_
+- `pro detach`_
+- `pro disable`_
+- `pro enable`_
+- `pro fix`_
+- `pro help`_
+- `pro refresh`_
+- `pro security-status`_
+- `pro status`_
+- `pro system`_
+
+
+pro api
+=======
+
+**Usage:**
+
+``pro api [-h] [--show-progress] [--args [OPTIONS ...]] [--data DATA] endpoint``
```

Comment:
Throughout the doc, we have the long name of all the options except -h and -o OPTION
Would it be better to standardise on the long form for consistency?

Reply:
It would. This will be part of future effort to always prefer long options in the CLI.
`-h` is quite an exception because argparse does support `--help` but only advertises the `-h`.

Reply:
That sounds sensible, happy to leave this as future work :)